ouroboros-network/src/Ouroboros/Network/TxSubmission/Inbound.hs

```diff
@@ -63,7 +63,11 @@ data TxSubmissionMempoolWriter txid tx idx m =
        mempoolAddTxs :: [tx] -> m [txid]
     }
 
-data TraceTxSubmissionInbound txid tx = TraceTxSubmissionInbound --TODO
+data TraceTxSubmissionInbound txid tx
+  = TraceTxSubmissions
+    { traceTxsCollected :: !Int
```

Comment:
Please change it to the following, to match the style in this module.

```
data TraceTxSubmissionInbound txid tx =
    TraceTxSubmissions {
      traceTxsCollected :: !Int,
      traceTxsWritten   :: !Int
    }
  deriving Show
```

Thanks.